js/google_ima.js

```diff
@@ -1264,6 +1297,8 @@ require("../html5-common/js/utils/utils.js");
         {
           //destroy the current ad manager is there is one
           _IMA_SDK_destroyAdsManager();
+          this.currentIMAAd = null;
```

Comment:
Does this mean that there are cases in which we want to destroy the AdsManager but also keep the current ads?